scimono-server/src/main/java/com/sap/scimono/callback/resourcetype/ResourceTypesCallback.java

```diff
@@ -19,4 +22,11 @@
    * @return the custom resource type or null if it does not exist
    */
   ResourceType getCustomResourceType(final String typeId);
+
+  /**
+   * Return the custom schema extensions for the specified resource type id.
+   * @param resourceTypeId
+   * @return list of custom schema extensions
+   */
+  List<SchemaExtension> getSchemaExtensions(final String resourceTypeId);
```

Comment:
It will be better to return default values (emptyList) for backwards compatibility.

Reply:
Absolutely! Done.